build.js

```diff
@@ -0,0 +1,12 @@
+var buildify = require('buildify');
+
+buildify()
+  .load('src/impress.js')
+  .concat(['src/plugins/navigation/navigation.js',
+           'src/plugins/resize/resize.js'])
+  .save('js/impress.js');
+/*
+ * Disabled until uglify supports ES6: https://github.com/mishoo/UglifyJS2/issues/448
```

Comment:
There is now `uglify-es` module for ES6 support, so we would need to switch to that to support it.

Not that it needs to be done in this PR, but just a note for future.

Reply:
Cool. I'll keep that in mind as a future improvement. I really like to provide the minified experience, but felt that not adopting language improvements was eventually holding back progress too much.